### ISSUE TITLE
Crash in Focus Plus when scrolling pages and switching hands.

### ISSUE THREAD

**Issue by daoshengmu** [MozillaReality/FirefoxReality#1214]
## Hardware

Focus Plus

## Steps to Reproduce

1. Switching hands and scrolling continuously at the feeding page.

## Current Behavior

Crash 

## Expected Behavior

Scrolling pages should work properly.

## Error Logs and Stack Traces

<details open>

```
2019-05-10 15:32:51.065 3763-3950/org.mozilla.vrbrowser I/VRActivityEventManager: update isFocused = 1, deviceID = 2, focusedController = 2
2019-05-10 15:32:51.065 3763-3950/org.mozilla.vrbrowser D/VROverlaySMD: onButtonEvent() eventType = TouchTapped inputId = Touchpad
2019-05-10 15:32:51.067 3763-3950/org.mozilla.vrbrowser D/VROverlaySMD: onButtonEvent() type = 3002 id = 16
2019-05-10 15:32:51.089 3763-3763/org.mozilla.vrbrowser W/System.err: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.view.InputDevice.getSources()' on a null object reference
2019-05-10 15:32:51.090 3763-3763/? W/System.err:     at org.mozilla.vrbrowser.ui.widgets.WindowWidget.handleHoverEvent(WindowWidget.java:374)
2019-05-10 15:32:51.091 3763-3763/? W/System.err:     at org.mozilla.vrbrowser.input.MotionEventGenerator.generateEvent(MotionEventGenerator.java:62)
2019-05-10 15:32:51.091 3763-3763/? W/System.err:     at org.mozilla.vrbrowser.input.MotionEventGenerator.dispatchScroll(MotionEventGenerator.java:130)
2019-05-10 15:32:51.091 3763-3763/? W/System.err:     at org.mozilla.vrbrowser.VRBrowserActivity.lambda$handleScrollEvent$20$VRBrowserActivity(VRBrowserActivity.java:619)
2019-05-10 15:32:51.091 3763-3763/? W/System.err:     at org.mozilla.vrbrowser.-$$Lambda$VRBrowserActivity$vQHdJe3f99DSbQ58Pcc4JBOLnQ4.run(lambda)
```

```
```

</details>

**Comment by daoshengmu** [comment]
@bluemarvin Already has a fix about this issue.

**Comment by bluemarvin** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1550901

**Comment by philip-lamb** [comment]
Fixed in #1222
